test/re_test.cc

```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
 
 #include <gtest/gtest.h>
-#include "re.h"
+#include "../src/re.h"
```

Comment:
I'll open an issues. Also benchmark_test.cc already sort of tests the regex matching by checking how many benchmarks were run given a specific regex.